src/engine/ExecuteUpdate.cpp

```diff
@@ -176,21 +176,28 @@ ExecuteUpdate::computeGraphUpdateQuads(
       cancellationHandle->throwIfCancelled();
     }
   }
-  auto sortAndRemoveDuplicates = [](auto& vec) {
-    ql::ranges::sort(vec);
-    vec.erase(std::unique(vec.begin(), vec.end()), vec.end());
-  };
   sortAndRemoveDuplicates(toInsert);
   sortAndRemoveDuplicates(toDelete);
-  metadata.inUpdate = DeltaTriplesCount{static_cast<int64_t>(toInsert.size()),
-                                        static_cast<int64_t>(toDelete.size())};
-  std::vector<IdTriple<>> reducedToDelete;
-  ql::ranges::set_difference(std::move(toDelete), toInsert,
-                             std::back_inserter(reducedToDelete));
-  toDelete.swap(reducedToDelete);
+  metadata.inUpdate_ = DeltaTriplesCount{static_cast<int64_t>(toInsert.size()),
+                                         static_cast<int64_t>(toDelete.size())};
+  toDelete = setMinus(toDelete, toInsert);
```

Comment:
Am I seeing this correctly that with the additional metadata returned from thhe DeltaTriples we can deduce the effect size of the `setMinus` function?